packages/react/src/components/UIShell/components/styles/_side-nav.scss

```diff
@@ -29,6 +29,45 @@ $prefix: 'cds' !default;
 }
 
 //----------------------------------------------------------------------------
+// Treeview Side-nav
+//----------------------------------------------------------------------------
+.#{$prefix}--side-nav__icon:not(.#{$prefix}--side-nav__submenu-chevron) {
+  margin-inline-end: $spacing-05;
+}
+
+.#{$prefix}--side-nav__submenu.#{$prefix}--side-nav__submenu--active {
+  > span {
+    color: $text-primary;
+    font-weight: 600;
+  }
+
+  .#{$prefix}--side-nav__submenu-chevron > svg {
+    fill: $icon-primary;
+  }
+}
+
+// Possibly dynamically do the thingie
+// .cds--side-nav__item {
+//   --depth: 0;
+
+//   .cds--side-nav__submenu {
+//     padding-left: calc(4.5rem * var(--depth));
+//   }
+
+//   .cds--side-nav__menu .cds--side-nav__menu-item .cds--side-nav__link {
+//     padding-left: calc(4.5rem * var(--depth) + 1rem);
+//   }
+// }
+
+// .cds--side-nav__submenu {
+//   --depth: 0;
+//   padding-left: calc(4.5rem * var(--depth));
+
+//   .cds--side-nav__link {
+//     padding-left: 1rem;
+//     background-color: red;
+//   }
+// }
```

Comment:
should we remove this or do we potentially need this?

Reply:
I'll remove it, its still in Ignacio's PR if he needs it